typescript-test-samples/lambda-handler-dynamodb/README.md

```diff
@@ -0,0 +1,121 @@
+
+## About this Pattern
+
+### System Under Test (SUT)
+
+The SUT in this pattern is an AWS Lambda function that makes calls to other AWS cloud services using an AWS SDK. For this example, we demonstrate a system that generates a new item, written to an Amazon DynamoDB table by the Lambda function.
+
+![System Under Test (SUT)](img/system-under-test.png)
+
+### Goal
+
+This pattern is intended to increase development speed. It enables rapid development and testing of a Lambda function that makes calls to other AWS cloud services using an AWS SDK. This pattern eliminates the need to perform a build and deploy of the Lambda function to the cloud between modifications of test or function code.
+
+### Description
+
+In this pattern you develop a Lambda function that makes calls to Amazon DynamoDB using an AWS SDK. Your tests invoke the handler function method on your local desktop, passing it a synthetic event as a parameter. The handler function executes within your local desktop environment. When the SDK invokes the Amazon DynamoDB table, it uses the [AWS credentials](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html) contained in your local environment, typically as environment variables. The SDK calls resources deployed to the cloud.
+
+In this sample, we have used AWS CDK (Cloud Development Kit) for infrastructure as code but the pattern can be also be used with a variety of systems including SAM, Serverless Framework, CloudFormation and Terraform. This pattern simplifies the process of attaching a debugger to the handler code. You may generate synthetic events using several methods including the ‘sam local generate-event’ command, by copying event template samples in the AWS Lambda console test section, or by printing events to CloudWatch logs.
+
+![Test Pattern](img/pattern_02_lambda_handler_test.png)
+
+### Limitations
+
+The IAM policies contained in your local environment variables may differ from those contained in the execution role attached to the Lambda function in the AWS cloud environment. Therefore this pattern may not accurately test IAM permissions. You may mitigate this risk by using the [IAM Roles Anywhere](https://docs.aws.amazon.com/rolesanywhere/latest/userguide/introduction.html) service to obtain temporary security credentials for the same execution role that the Lambda function uses in the cloud context. This service is open source and you may [view the code on GitHub](https://github.com/aws/rolesanywhere-credential-helper).
+
+[Top](#contents)
+---
+
+## About this Example
+
+This example contains an [AWS Lambda](https://aws.amazon.com/lambda/) function and [Amazon DynamoDB](https://aws.amazon.com/dynamodb/) table as core resources.
+
+AWS CDK is used to deploy the AWS Lambda function and Amazon DynamoDB table. The AWS Lambda function generates an item in the JSON format and writes this item to the DynamoDB table using the AWS SDK. The DynamoDB table item is a JSON object with format:
+
+```json
+{
```

Comment:
This doesn't seem to be the right item structure. It should match what the handler is writing in `app.ts` on line 22+.

Reply:
Updated the json structure in ReadMe. Please review.